ingest/exporter/ingestexportservice.py

```diff
@@ -399,17 +402,16 @@ def bundle_links(self, links):
     def upload_metadata_files(self, submission_uuid, metadata_files_info):
         try:
             for metadata_type in ['project', 'biomaterial', 'process', 'protocol', 'file', 'links']:
-                for metadata_doc in metadata_files_info[metadata_type]:
-                    bundle_file = metadata_doc
-                    filename = bundle_file['upload_filename']
-                    content = bundle_file['content']
-                    content_type = bundle_file['content_type']
-
-                    if not bundle_file.get('is_from_input_bundle'):
-                        uploaded_file = self.upload_file(submission_uuid, filename, content, content_type)
-                        bundle_file['upload_file_url'] = uploaded_file.url
-        except Exception as e:
-            message = "An error occurred on uploading bundle files: " + str(e)
+                bundle_file = metadata_files_info[metadata_type]
```

Comment:
I think the for loop was removed by accident here? `metadata_files_info[metadata_type]` evals to an array (`metadata_files_info` is a map/dict of the above `metadata_types` to arrays of metadata documents of that type)

Reply:
Great catch! Fixed!